.ruby-version

```diff
@@ -1 +1 @@
-2.3.5
+2.5.3
```

Comment:
This should match the version in `.travis.yml`. Let's use the most recent 2.5 version, 2.5.7.

Reply:
ok I was trying to keep it consistent with our other apps.